lxd/db_images.go

```diff
@@ -105,8 +105,17 @@ func dbImageSourceGet(db *sql.DB, imageId int) (int, api.ImageSource, error) {
 
 }
 
-// dbImageGet gets an ImageBaseInfo object from the database.
-// The argument fingerprint will be queried with a LIKE query, means you can
+// Whether an image with the given fingerprint exists.
+func dbImageExists(db *sql.DB, fingerprint string) (exists bool, err error) {
```

Comment:
Then I'm not sure what's the difference between your code and mine, beside the named values :)
My code also returns the boolean (which is set via outargs by dbQueryRowScan() and the err, returned by the same function.
It's effectively the same as:

```
query := "SELECT COUNT(*) > 0 FROM images WHERE fingerprint=?"
inargs := []interface{}{fingerprint}
outargs := []interface{}{&exists}
err = dbQueryRowScan(db, query, inargs, outargs)
return exists, err
```